aten/src/ATen/Parallel.h

```diff
@@ -128,6 +128,9 @@ CAFFE2_API int get_num_interop_threads();
 
 // Launches inter-op parallel task
 CAFFE2_API void launch(std::function<void()> func);
+namespace internal {
+  void launch_no_thread_state(std::function<void()> fn);
```

Comment:
nit: align the `func` name with `launch`

Reply:
sorry, clash of different style guidelines in my head :) fixed.